api/VREyeParameters.json

```diff
@@ -0,0 +1,165 @@
+{
+  "version": "1.0.0",
+  "data": {
+    "api": {
+      "VREyeParameters": {
```

Comment:
Here the obsolete methods `maximumFieldOfView`, `minimumFieldOfView`, `recommendedFieldOfView` and `renderRect` (with a status object) are missing. Are they implemented anywhere? Removed from the spec? Relevant to anyone? I think if they are on MDN they are relevant to some people (with legacy code) and so the compat data should also signal them that this is deprecated and no longer supported. If this isn't the case, they shouldn't be on MDN and also not in here. Does that make sense to you?